dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -827,16 +829,18 @@ private boolean handleRightLabelEqualityRelationshipTypeElement(Context c, Entit
      *                                          for the left entity
      * @param relationshipType                  The RelationshipType object that we want to check whether it's
      *                                          valid to be added or not
-     * @return                                  A boolean indicating whether the relationship is left or right, will
+     * @return A boolean indicating whether the relationship is left or right, will
      *                                          be true in this case
      * @throws SQLException                     If something goes wrong
      */
-    private boolean handleLeftLabelEqualityRelationshipTypeElement(Context c, Entity entity, Entity relationEntity,
-                                                                   boolean left,
-                                                                   List<RelationshipType> acceptableRelationshipTypes,
-                                                                   List<RelationshipType>
-                                                                       leftRelationshipTypesForEntity,
-                                                                   RelationshipType relationshipType)
+    private boolean handleLeftwardTypeNameEqualityRelationshipTypeElement(Context c, Entity entity,
```

Comment:
This method has exactly the same description as the previous one, adding to the mystery surrounding what they do.

Reply:
I've renamed the method to verifyValidLeftwardRelationshipType
The JavaDoc has been improved as well to explain what the method does